pliers/extractors/text.py

```diff
@@ -313,3 +332,114 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+
+class SpaCyTokenExtractor(TextExtractor):
+    '''
+    Uses SpaCy to extract features from text. Extracts features for every word(token) in a sentence.
+    Features returned are:
+     text : Verbatim text content.(unicode)
+     lemma_ : Base form of the token, with no inflectional suffixes.(unicode)
+     pos_ : Coarse-grained part-of-speech.(unicode)
+     tag_ : Fine-grained part-of-speech.(unicode)
+     dep_ : Syntactic dependency relation.(unicode)
+     shape_ : Transform of the tokens's string, to show orthographic features. For example, "Xxxx" or "dd".(unicode)
+     is_alpha : Does the token consist of alphabetic characters? (bool)
+     is_stop : Is the token part of a "stop list"? (bool)
+     is_punct : Is the token punctuation?(bool)
+     sentiment : A scalar value indicating the positivity or negativity of the token.(float)
+     is_ascii : Does the token consist of ASCII characters?(bool)
+     is_digit : Does the token consist of digits?(bool)
+    '''
```

Comment:
The docstring should also have a separate `Args:` section that describes what the arguments (`model` and `features` are). See other `Extractors` for examples.